themes/classic/_dev/css/partials/_variables.scss

```diff
@@ -8,6 +8,7 @@ $brand-secondary: #f39d72;
 $brand-success: #4cbb6c;
 $brand-warning: #ff9a52;
 $brand-danger: #ff4c4c;
+$last-items: #ff9a52;
```

Comment:
@julienbourdeau @thierrymarianne Yes, please don't duplicate variable if it's the same color as $brand-warning